src/node/view_change.h

```diff
@@ -60,5 +60,30 @@ namespace ccf
 
       return v;
     }
+    MSGPACK_DEFINE(signatures, signature);
   };
+  DECLARE_JSON_TYPE(ViewChange);
+  DECLARE_JSON_REQUIRED_FIELDS(ViewChange, signatures, signature);
+
+  struct NewView
```

Comment:
imo this would be better named ViewChanges or ViewHistory, and ViewChange is really ViewChangeRequest or ViewChangeVote.

I think having new_view later is particularly confusing.

Reply:
thanks! I made it ViewChangeRequest and ViewChangeConfirmation. I think that does a good job of describing what is going on.